consts.js

```diff
@@ -0,0 +1,32 @@
+// The list of the default externals
+// https://webpack.js.org/configuration/externals/
+const defaultExternals = [
+  '@folio/stripes',
+  '@folio/stripes-components',
+  '@folio/stripes-connect',
+  '@folio/stripes-core',
+  '@folio/stripes-util',
+  '@folio/stripes-form',
+  '@folio/stripes-final-form',
+  '@folio/stripes-logger',
+  '@folio/stripes-smart-components',
+  'final-form',
+  'final-form-arrays',
+  'moment',
+  'moment-timezone',
```

Comment:
`moment-timezone` isn't currently provided by the platform for apps use as a peer-dep but I think it needs to be for this to work correctly. After this PR, it would be extern'ed in apps but also never pulled in by the platform, and thus absent in the final bundle. (Right?)